src/loader.ts

```diff
@@ -55,8 +59,17 @@ export default function loader(
   // "export default" on the page
   if (!defaultExport) return rawCode
 
-  if (isAppDirNext13) {
-    return templateAppDir(pagePkg, { hasLoadLocaleFrom, pageNoExt, normalizedResourcePath, normalizedPagesPath })
+  if (normalizedPagesPath.endsWith('app/')) {
+    if (page.endsWith('page.tsx')) {
```

Comment:
Here we need to take into account all the possible formats (.tsx, .js, .md, etc...)